packages/settings.gradle.kts

```diff
@@ -24,6 +24,7 @@ dependencyResolutionManagement {
         getPropertyValue("testRepository")?.let {
             maven("file://${rootDir.absolutePath}/$it")
         }
+        maven("https://oss.sonatype.org/content/repositories/snapshots")
```

Comment:
I am a bit hesitant to leave this here, as you might end up compiling stuff against public snapshots instead of locally distributed once. Do we really need it, or could we just have it commented out for easy access when needed.